### PR TITLE
[JENKINS-52068] Remove reflection and code duplication

### PR DESCRIPTION
**Pull request by jonahgraham**
This came up when working on the Java 9/10/11 hackathon. See https://gitter.im/jenkinsci/jenkins?at=5b2a2b1772b31d3691dc0c15

### CODE REVIEW

**Comment by oleg-nenashev** [comment]
@jenkinsci/java10-support

File: src/main/java/org/jenkinsci/plugins/parameterizedscheduler/FieldAccessor.java [hunk 1]
```diff
@@ -1,29 +0,0 @@
...
-
-import java.lang.reflect.Field;
-
-/**
- * lets work around some final classes I couldn't subclass, but don't want to duplicate everything
- * 
- * @author jameswilson
- *
- */
-public class FieldAccessor {
```
Comment (oleg-nenashev) [inline src/main/java/org/jenkinsci/plugins/parameterizedscheduler/FieldAccessor.java]: Formally it breaks Binary compatibility.
There is no usages outside the plugin, so it should be fine in this case

File: src/main/java/org/jenkinsci/plugins/parameterizedscheduler/ParameterizedCronTab.java [hunk 2]
```diff
@@ -22,22 +19,14 @@
 public class ParameterizedCronTab {
 
 	private final Map<String, String> parameterValues;
-	private final long[] bits;
-	private final Integer dayOfWeek;
-	private final CronTab cronTab;
```
Comment (oleg-nenashev) [inline src/main/java/org/jenkinsci/plugins/parameterizedscheduler/ParameterizedCronTab.java]: This class is not persisted on the disk: https://github.com/jenkinsci/parameterized-scheduler-plugin/blob/4a47537d58f4f5a0b528ba710a4f849bcf3459bf/src/main/java/org/jenkinsci/plugins/parameterizedscheduler/ParameterizedTimerTrigger.java#L30 , so such cleanup should be fine

**Comment by batmat** [comment]
Released as 0.6.2, thanks !

**Comment by oleg-nenashev** [comment]
Thanks @batmat !

**Pull request merged**